internal/services/network/network_manager_ipam_pool_resource_test.go

```diff
@@ -0,0 +1,246 @@
+
+func testAccNetorkManagerIpamPool_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_network_manager_ipam_pool", "test")
+	r := ManagerIpamPoolResource{}
+
+	data.ResourceSequentialTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+	})
+}
+
+func testAccNetorkManagerIpamPool_basicIPv6(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_network_manager_ipam_pool", "test")
+	r := ManagerIpamPoolResource{}
+
+	data.ResourceSequentialTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.basicIpv6(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+	})
+}
+
+func testAccNetorkManagerIpamPool_update(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_network_manager_ipam_pool", "test")
+	r := ManagerIpamPoolResource{}
+
+	data.ResourceSequentialTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+		{
+			Config: r.update(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+	})
+}
+
+func testAccNetorkManagerIpamPool_requiresImport(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_network_manager_ipam_pool", "test")
+	r := ManagerIpamPoolResource{}
+
+	data.ResourceSequentialTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.RequiresImportErrorStep(r.requiresImport),
+	})
+}
+
+func testAccNetorkManagerIpamPool_complete(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_network_manager_ipam_pool", "test")
+	r := ManagerIpamPoolResource{}
+
+	data.ResourceSequentialTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.complete(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+	})
+}
+
+func (r ManagerIpamPoolResource) Exists(ctx context.Context, client *clients.Client, state *pluginsdk.InstanceState) (*bool, error) {
+	id, err := ipampools.ParseIPamPoolID(state.ID)
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := client.Network.IPamPools.Get(ctx, *id)
+	if err != nil {
+		return nil, fmt.Errorf("retrieving %s: %+v", *id, err)
+	}
+
+	return pointer.To(resp.Model != nil), nil
+}
+
+func (r ManagerIpamPoolResource) basic(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+%[1]s
+
+provider "azurerm" {
+  features {}
+}
```

Comment:
For consistency, can we flip these in all of the tests?

```suggestion
provider "azurerm" {
  features {}
}

%[1]s
```

Reply:
fixed